src/positioning/CssTranslatePositionProvider.js

```diff
@@ -41,19 +41,17 @@ class CssTranslatePositionProvider extends PositionProvider {
    * @override
    * @inheritDoc
    */
-  setPosition(location) {
+  async setPosition(location) {
     ArgumentGuard.notNull(location, "location");
 
-    const that = this;
     this._logger.verbose(`CssTranslatePositionProvider - Setting position to: ${location}`);
 
     const spp = new ScrollPositionProvider(this._logger, this._executor);
```

Comment:
Why do you need this? This method looks much more complicated than needed.
Check my JS version or [Java one](https://github.com/applitools/eyes.sdk.java3/blob/846213f1b1a54bb096acf0e6f936a1e25a9ceea7/eyes.selenium.java/src/main/java/com/applitools/eyes/selenium/positioning/CssTranslatePositionProvider.java#L44).

Reply:
A complete change to this method according to java is not related to the task. We need just make a workaround for chrome issue.

Reply:
Then your changes should go to `EyesWDIOUtils.translateTo` according to your implementation this method moves the page to a position.
So, the fix should be inside the method. Maybe you use it from other places as well and they will require the fix too.

Reply:
moved changes into EyesWDIOUtils.translateTo